CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ else()
   add_definitions(-D_WIN32_WINNT=0x600)
   set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267 /wd4244")
+  add_compile_options("$<$<NOT:$<CONFIG:Debug>>:/Zi>")
```

Comment:
is this required?

Reply:
Added debug symbols as part of another PR